Modules/_datetimemodule.c

```diff
@@ -6537,136 +6514,127 @@ PyInit__datetime(void)
     };
 
     for (size_t i = 0; i < Py_ARRAY_LENGTH(types); i++) {
-        if (PyModule_AddType(m, types[i]) < 0) {
-            return NULL;
+        if (PyModule_AddType(module, types[i]) < 0) {
+            return -1;
         }
     }
 
     if (PyType_Ready(&PyDateTime_IsoCalendarDateType) < 0) {
-        return NULL;
+        return -1;
     }
     Py_INCREF(&PyDateTime_IsoCalendarDateType);
```

Comment:
I don't understand the purpose of this INCREF? Can you try to remove it and check that the test_datetime doesn't crash?

Reply:
@vstinner confirmed